blaze-client/src/main/scala/org/http4s/client/blaze/BlazeClientBuilder.scala

```diff
@@ -293,7 +312,7 @@ object BlazeClientBuilder {
       maxTotalConnections = 10,
       maxWaitQueueLimit = 256,
       maxConnectionsPerRequestKey = Function.const(256),
-      sslContext = sslContext,
+      sslContext = toMaybeSSLContext(sslContextOption),
```

Comment:
And then the type of this value turns into the new `SSLContextOption`.

Reply:
I believe that I've made the necessary changes to use `SSLContextOption`. Please take a look at https://github.com/http4s/http4s/pull/3422#commits-pushed-9c170ef, @rossabaker.